content-services/7.3/support/index.md

```diff
@@ -83,7 +192,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | Identity Service 1.8 | |
 | Alfresco Intelligence Services 1.4.5 | |
 | Alfresco Content Connector for AWS S3 5.1 | Adds support for AWS Glacier using Cloud storage layer. |
-| Alfresco Content Connector for Azure 3.1 | |
+| Alfresco Content Connector for Azure 3.1.1 | |

```

Comment:
Only ACS 7.2.2 has been tested with Azure 3.1.1, so I think this row & the next one should be left as they were before - Azure 3.1 (for 7.3.1 & 7.3.0).